sycl/test/matrix/matrix-int8-test.cpp

```diff
@@ -1,8 +1,11 @@
 // RUN: %clangxx -fsycl -fsycl-device-only -O2 -S -emit-llvm -o - %s | FileCheck %s
 
-// CHECK-DAG: %"struct.__spv::__spirv_JointMatrixINTEL" = type { [12 x [48 x [1 x [4 x i8]]]] addrspace(4)* }
-// CHECK-DAG: %"struct.__spv::__spirv_JointMatrixINTEL.[[#]]" = type { [12 x [12 x [1 x [4 x i32]]]] addrspace(4)* }
-// CHECK-DAG: %"struct.__spv::__spirv_JointMatrixINTEL.[[#]]" = type { [48 x [12 x [4 x [4 x i8]]]] addrspace(4)* }
+// CHECK-DAG: %"struct.__spv::__spirv_JointMatrixINTEL" =
+// CHECK-DAG: type { [12 x [48 x [1 x [4 x i8]]]] addrspace(4)* }
+// CHECK-DAG: %"struct.__spv::__spirv_JointMatrixINTEL.[[#]]" =
+// CHECK-DAG: type { [12 x [12 x [1 x [4 x i32]]]] addrspace(4)* }
+// CHECK-DAG: %"struct.__spv::__spirv_JointMatrixINTEL.[[#]]" =
+// CHECK-DAG: type { [48 x [12 x [4 x [4 x i8]]]] addrspace(4)* }
```

Comment:
This is not the right way to handle clang-format issues.
We agreed ignore FileCheck comments, so you should update https://github.com/intel/llvm/blob/sycl/sycl/test/.clang-format to ignore `CHECK-DAG: `.